<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogShellMenu</class>
 <widget class="QDialog" name="DialogShellMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>688</width>
    <height>556</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Shell Menu</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="buttonNew">
           <property name="text">
            <string>New</string>
           </property>
           <property name="icon">
            <iconset theme="document-new">
             <normaloff>.</normaloff>.</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="buttonDelete">
           <property name="text">
            <string>Delete</string>
           </property>
           <property name="icon">
            <iconset theme="edit-delete">
             <normaloff>.</normaloff>.</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="buttonCopy">
           <property name="text">
            <string>Copy</string>
           </property>
           <property name="icon">
            <iconset theme="edit-copy">
             <normaloff>.</normaloff>.</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="buttonUp">
           <property name="text">
            <string>Move</string>
           </property>
           <property name="icon">
            <iconset theme="go-up">
             <normaloff>.</normaloff>.</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="buttonDown">
           <property name="text">
            <string>Move</string>
           </property>
           <property name="icon">
            <iconset theme="go-down">
             <normaloff>.</normaloff>.</iconset>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QListView" name="listItems"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Select a shell menu item from the list at left.
Select &quot;New&quot; to add a new command to the menu.</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Menu Entr&amp;y</string>
           </property>
           <property name="buddy">
            <cstring>editName</cstring>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>(&gt; for sub-menu, @ language mode)</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLineEdit" name="editName">
         <property name="clearButtonEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Acce&amp;lerator</string>
             </property>
             <property name="buddy">
              <cstring>editAccelerator</cstring>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>158</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="1" column="0" colspan="2">
            <widget class="KeySequenceEdit" name="editAccelerator"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Command Input (stdin):</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QRadioButton" name="radioFromSelection">
            <property name="text">
             <string>&amp;selection</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioFromDocument">
            <property name="text">
             <string>document</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioFromEither">
            <property name="text">
             <string>e&amp;ither</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioFromNone">
            <property name="text">
             <string>n&amp;one</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Command Output (stdout/stderr)</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QRadioButton" name="radioToSameDocument">
            <property name="text">
             <string>sa&amp;me document</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioToDialog">
            <property name="text">
             <string>dialo&amp;g</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioToNewDocument">
            <property name="text">
             <string>&amp;new document</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkReplaceInput">
         <property name="text">
          <string>Output replaces input</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkSaveBeforeExec">
         <property name="text">
          <string>Save &amp;file before executing command</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkReloadAfterExec">
         <property name="text">
          <string>&amp;Re-load file after executing command</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Shell Command to E&amp;xecute</string>
       </property>
       <property name="buddy">
        <cstring>editCommand</cstring>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>428</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>(% expands to current filename, # to line number)</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="editCommand">
     <property name="font">
      <font>
       <family>Monospace</family>
      </font>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Close|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KeySequenceEdit</class>
   <extends>QLineEdit</extends>
   <header>KeySequenceEdit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogShellMenu</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>469</x>
     <y>620</y>
    </hint>
    <hint type="destinationlabel">
     <x>505</x>
     <y>599</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
